mantaray/backend.py

```diff
@@ -245,6 +255,7 @@ def run_one_step(self) -> None:
 
             self._ping_sent = False
             self._last_receive_time = time.monotonic()
+            self._nickmask = None
```

Comment:
Why is this here, instead of under the "Clearing data from previous connections" comment?